examples/semantic_segmentation/semantic_segmentation/data_loader.py

```diff
@@ -52,6 +52,7 @@ def load_batch(
         def load(ts: TestSample, gt: GroundTruth, inf: Inference) -> Tuple[str, np.ndarray, np.ndarray]:
             inf_prob = download_binary_array(inf.prob.locator)
             gt_mask = download_mask(gt.mask.locator)
+            gt_mask[gt_mask != 1] = 0  # binarize gt_mask
```

Comment:
The issue with the incorrect recall values was because the gt mask that we use includes other class as well.  Therefore when computing FN mask, it copies GT mask where GT mask != INF mask which threw off the FN counts. The FN masks looked fine on the frontend because it only shows pixels with label 1: person.